examples/sandbox/package.json

```diff
@@ -21,7 +21,7 @@
     "@shapeshiftoss/hdwallet-ledger": "1.55.9",
     "@shapeshiftoss/hdwallet-ledger-webhid": "1.55.9",
     "@shapeshiftoss/hdwallet-ledger-webusb": "1.55.9",
-    "@shapeshiftoss/hdwallet-metamask": "1.55.9",
+    "@shapeshiftoss/hdwallet-metamask-multichain": "1.55.10-mipd.0",
```

Comment:
Sandbox is a big mess - as far as I can understand from these stale refs, this doesn't use lerna but upstream (where `@shapeshiftoss/hdwallet-metamask-multichain` only exists as `1.55.10-mipd.0` alpha for now) , and trying to use 1.55.10 across the board here is a rug:
<img width="847" alt="image" src="https://github.com/user-attachments/assets/df28f564-01b5-4eb3-b80f-e3d0bc703c55">

So looks like we're always one version behind in this package.json pretty much?

Reply:
Actually shadowed *is* fine, that's what we want here not to always be one version behind and have to manually update: 7309f547
<img width="616" alt="image" src="https://github.com/user-attachments/assets/fdcb9730-89cd-48b3-82c6-62bd4885d100">

Reply:
Actually, shadowed *is* fine and that's the reason why things were borked in the first place with this package rename, and why we were always a version behind and had to manually update. 7309f547e9efc247c128c560c64ea1fe19dcf1a4